src/Roles.js

```diff
@@ -150,7 +150,7 @@ const ROLE_TAG_MAPPING = {
   searchbox: 'input',
   slider: 'input',
   spinbutton: 'input',
-  'switch': 'button',
+  switch: 'button',
```

Comment:
Having the quotes here allows for syntax highlighting to work correctly since `switch` is a keyword in js.  Please disable the particular linting rule for this line instead.

Reply:
updated